### PR TITLE
[REVIEW] Update README and some docs

### PR DESCRIPTION
**Pull request by BradReesWork**
Update the README with new algorithms and multi-gpu status
Also cleaned up a few other stops

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by BradReesWork** [comment]
rerun tests

**Comment by BradReesWork** [comment]
rerun tests

**Comment by BradReesWork** [comment]
rerun tests

File: README.md [hunk 1]
```diff
@@ -57,25 +57,47 @@ for i in range(len(df_page)):
...
+* cuDF DataFrame
+* Pandas DataFrame
+
+cuGraph supports execution of graph algorithms from different graph objects
+* cuGraph Graph classes
+* NetworkX graph classes
+* CuPy sparse matrix
+* SciPy sparse matrix
+
+cuGraph tries to match the return type based on the input type.  So a NetworkX input will return the same data type that Network would have.
```
Comment (afender) [inline README.md:100]: Network X

**Review by afender — APPROVED** [review]
Looks good, pending CI checks passing

File: README.md [hunk 2]
```diff
@@ -57,25 +57,47 @@ for i in range(len(df_page)):
...
+|              | Single Source Shortest Path (SSSP)     | Multi-GPU    |                     |
 | Structure    |                                        |              |                     |
-|              | Renumbering                            | Single-GPU   | Also for multiple columns  |
-|              | Symmetrize                             | Single-GPU   |                     |
-
+|              | Renumbering                            | Single-GPU   | multiple columns, any data type  |
+|              | Symmetrize                             | Multi-GPU    |                     |
+| Other        |                                        |              |                     |
+|              | Hungarian Algorithm                    | Single-GPU   |                     |
+|              | Minimum Spanning Tree                  | Single-GPU   |                     |
```
Comment (afender) [inline README.md:75]:
```suggestion
|              | Minimum Spanning Tree                  | Single-GPU   |                     |
|              | Maximum Spanning Tree                  | Single-GPU   |                     |

```

File: README.md [hunk 3]
```diff
@@ -57,25 +57,47 @@ for i in range(len(df_page)):
...
-|              | Single Source Shortest Path (SSSP)     | Multiple-GPU |                     |
+|              | Breadth First Search (BFS)             | Multi-GPU    |                     |
+|              | Single Source Shortest Path (SSSP)     | Multi-GPU    |                     |
 | Structure    |                                        |              |                     |
-|              | Renumbering                            | Single-GPU   | Also for multiple columns  |
-|              | Symmetrize                             | Single-GPU   |                     |
-
+|              | Renumbering                            | Single-GPU   | multiple columns, any data type  |
+|              | Symmetrize                             | Multi-GPU    |                     |
+| Other        |                                        |              |                     |
```
Comment (afender) [inline README.md:73]: Should spanning-tree algos in a `Tree` category instead of `Others`?

File: README.md (hunk 2 above)
Comment (afender) [reply README.md:75]: Nx lists it as two separate features ...

**Review by afender — APPROVED** [review]
Just a few more mst related suggestions

File: README.md (hunk 3 above)
Comment (BradReesWork) [reply README.md:73]: once we have more Tree algos, then yes

**Comment by BradReesWork** [comment]
rerun tests

**Pull request merged**